### PR TITLE
Nick/ok2.0

### PR DESCRIPTION
**Pull request by nwadams**
@lxdvs thoughts?
we need to make sure the multipost thing isn't an issue though. I wasn't able to repro it with my emulator in charles in a few minutes of testing but I could be testing wrong. 
I know it manifested itself with big image uploads.

### CODE REVIEW

File: src/com/android/volley/toolbox/Volley.java [hunk 1]
```diff
@@ -45,17 +41,6 @@ public static RequestQueue newRequestQueue(Context context, HttpStack stack) {
 
         if (stack == null) {
             stack = new OkHttpStack();
-            OkHttpClient client = ((OkHttpStack)stack).getClient();
```
Comment (cdeonier) [inline src/com/android/volley/toolbox/Volley.java:48]: This block looks important. Is SSL set up somewhere else?

Comment (nwadams) [inline src/com/android/volley/toolbox/Volley.java:48]: okhttp does its own ssl setup normally.

**Comment by cdeonier** [comment]
:ship:

File: src/com/android/volley/toolbox/HurlStack.java [hunk 2]
```diff
@@ -103,12 +99,16 @@ public HttpResponse performRequest(Request<?> request, Map<String, String> addit
         }
         URL parsedUrl = new URL(url);
         HttpURLConnection connection = openConnection(parsedUrl, request);
-        
+
+        /*
+        not sure if this is needed with the okhttp-urlconnection wrapper
+        also these methods no longer exist
```
Comment (lxdvs) [inline src/com/android/volley/toolbox/HurlStack.java:105]: If this is working during testing and throttling conditions (ie not double uploading photos), lets just roll with it.

Comment (nwadams) [inline src/com/android/volley/toolbox/HurlStack.java:105]: i wasn't able to repro it yet. I'm going to keep trying things though.

File: src/com/android/volley/toolbox/Volley.java (hunk 1 above)
Comment (lxdvs) [inline src/com/android/volley/toolbox/Volley.java:48]: Yeah this was just for treatment of native crashes.

**Comment by lxdvs** [comment]
+1 :)

**Pull request merged**